helm-base-versions/helm-chart-base-versions.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+set -e
+
+[[ "$SCRIPT_DEBUG" == "true" ]] && set -x
+
+CLUSTER_NAME=${1:-$CLUSTER_NAME}
+ARTIFACT_URL=${2:-$ARTIFACT_URL}
+API_SERVER_URL=${3:-$API_SERVER_URL}
+COSMOS_KEY=${4:-$COSMOS_KEY}
```

Comment:
can we use managed identity auth please? or create a follow up ticket.
we aren't supposed to use key based auth and have previously told secops we only use AAD auth for our team and cosmos